doc/ch-image.rst

```diff
@@ -1000,11 +1005,660 @@ Features we do not plan to support
 
 * :code:`USER` does not make sense for unprivileged builds.
 
-* :code:`VOLUME`: This instruction is not currently supported. Charliecloud
+* :code:`VOLUME`: Charliecloud
   has good support for bind mounts; we anticipate that it will continue to
   focus on that and will not introduce the volume management features that
   Docker has.
 
+.. _ch-image_rsync:
+
+:code:`RSYNC` (Dockerfile extension)
+------------------------------------
+
+.. warning::
+
+   This instruction is experimental and may change or be removed.
+
+Overview
+~~~~~~~~
+
+Copying files is often simple but has numerous difficult corner cases, e.g.
+when dealing with symbolic or hard links. The standard instruction
+:code:`COPY` deals with many of these corner cases differently from other UNIX
+utilities, lacks complete documentation, and behaves inconsistently between
+different Dockerfile interpreters (e.g., Docker’s legacy builder vs.
+BuildKit), as detailed above. On the other hand, :code:`rsync(1)` is an
+extremely capable, widely used file copy tool, with detailed options to
+specify behavior and 25 years of history dealing with weirdness.
+
+:code:`RSYNC` (also spelled :code:`NSYNC`) is a Charliecloud extension that
```

Comment:
Can I really use NSYNC in my Dockerfile!? (love it).

Reply:
Yes you can!